lib/apiservers/engine/backends/archive.go

```diff
@@ -636,12 +643,34 @@ func (rm *ArchiveStreamReaderMap) Close() {
 	rm.prefixTrie.Visit(closeStream)
 }
 
+// try to fake the statpath for path that targets the mountpoint or along the mountpoint
+func tryFakeStatPath(mounts []types.MountPoint, target string) (*types.ContainerPathStat, bool) {
+	isMountPathTarget := false
+
+	for _, mount := range mounts {
+		if strings.HasPrefix(mount.Destination, target) {
```

Comment:
Are you sure this is correct?  What if mount has '/mnt/A' and target is '/mnt'?  If I understand this logic right, I think you're looking for an exact match.